effortless/effortless.py

```diff
@@ -343,6 +362,7 @@ def add(self, entry: dict) -> None:
         data["content"].append(entry)
         self._write_db(data)
         self._handle_backup()
+        logger.debug(f"Entry added to database: {entry}")
```

Comment:
This seems to generate significant performance overhead when large changes are made -- I would suggest truncating the message to a maximum length, which I am hoping will stop the unit tests from hanging. Once the unit tests pass, I'll merge these changes in!

Reply:
Yes I'll look into it.